.editorconfig

```diff
@@ -0,0 +1,17 @@
+; http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{go,py,rb,php}]
```

Comment:
I think you took an earlier `.editorconfig` file and just pasted here. No problem. Just don't believe whatever others tell you.

Reply:
Yes, pasted this file before main code. I think this file similar to ```.gitignore``` - possible to just past and its not bad practise